zerocom/utils/log.py

```diff
@@ -25,21 +19,12 @@ def setup_logging() -> None:
     root_log = logging.getLogger()
     log_formatter = logging.Formatter(LOG_FORMAT)
 
-    if coloredlogs is not None:
-        if "COLOREDLOGS_LOG_FORMAT" not in os.environ:
-            coloredlogs.DEFAULT_LOG_FORMAT = LOG_FORMAT
-
-        if "COLOREDLOGS_LEVEL_STYLES" not in os.environ:
-            coloredlogs.DEFAULT_LEVEL_STYLES = {
-                **coloredlogs.DEFAULT_LEVEL_STYLES,
-                "critical": {"background": "red"},
-            }
-
-        coloredlogs.install(level=logging.DEBUG, logger=root_log, stream=sys.stdout)
-    else:
-        stdout_handler = logging.StreamHandler(stream=sys.stdout)
-        stdout_handler.setFormatter(log_formatter)
-        root_log.addHandler(stdout_handler)
+    logging.basicConfig(
+        level=LOG_LEVEL,
+        format=LOG_FORMAT,
+        handlers=[RichHandler(show_time=False)],
+        datefmt="%Y-%m-%d %H:%M:%S",  # TODO: Add millisecond precision
+    )
```

Comment:
Why use basic config? We're already setting everything manually. The log level is set on line 34 and there's already a `log_formatter` crated with the format, so we should be able to just add that formatter for the rich handler, just like adding a formatter for any log handler and then add that handler to the root log directly.

```py
rich_handler = RichHandler(show_time=False)
rich_handler.setFormatter(log_formatter)
root_log.addHandler(rich_handler)
```